pom.xml

```diff
@@ -88,7 +88,7 @@
 
     <!-- commonjava/redhat projects -->
     <atlasVersion>1.1.1</atlasVersion>
-    <galleyVersion>1.9</galleyVersion>
+    <galleyVersion>2.0-SNAPSHOT</galleyVersion>
```

Comment:
I've updated galley to 1.10-SNAPSHOT for next development version. Please change this accordingly.